frontend/src/components/common/WeeklyEventItemPopUp.tsx

```diff
@@ -0,0 +1,211 @@
+import { CalendarIcon, TimeIcon } from "@chakra-ui/icons";
+import {
+  Badge,
+  Container,
+  Divider,
+  Flex,
+  Grid,
+  HStack,
+  Modal,

```

Comment:
Can u delete this file? It was moved inside Calender folder by Akshay in his pr.